README.md

```diff
@@ -101,7 +101,7 @@ The following are public methods of PageCache class that you could call from you
 - disableSession():void - Disable session support.
 - sessionExclude(array):void - Exclude $_SESSION key(s) from caching strategies 
 - isCached():bool - Checks if current page is in cache, true if exists false if not cached yet.
-- getFilePath():string - Get full path for current page's filename. At this point file itself might or might not have been created.
+- getFilepath():string - Get full path for current page's filename. At this point file itself might or might not have been created.
```

Comment:
Let's not change this. Filepath are two separate words, let's keep it getFilePath